Src/HostingStartup/HostingStartup.Net45.Tests/HostingStartup.Net45.Tests.csproj

```diff
@@ -0,0 +1,96 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'Test.props'))\Test.props" />
```

Comment:
this project should be onboarded on microbuild to have a proper signature in build machine. Otherwise you wouldn't be able to set `InternalsVisibleTo` when you need it